assets/_scss/default/body-main-styles.scss

```diff
@@ -226,21 +226,48 @@ ul.sub-item {
 }
 .photoTitle{
   display: flex;
-  flex-wrap: wrap;
   align-items: center;
+  margin-bottom: 1vw;
 }
 .containertitle {
-  display: flex;
-  flex-wrap: wrap;
-  align-items: center;
   text-align: center;
-  width: 65%;
-  margin: 0 auto;
+  padding: 1vw 0;
+}
+.even-background {
+  >:nth-child(odd) {
+    background-color: transparent;
+  }
+  >:nth-child(even) {
+    background-color: var(--even-background-color);
+  }
+}
+#opportunities {
```

Comment:
This should belong to a file for its own view. Please, review what can be moved from this file to others.